### PR TITLE
chore(pf5): upgrade Topology View to Patternfly 5

### PR DESCRIPTION
**Pull request by tthvo**
# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
* [x] Read the [contributing guidelines](https://github.com/cryostatio/cryostat/blob/main/CONTRIBUTING.md)
* [x] Linked a relevant issue which this PR resolves
* [x] Linked any other relevant issues, PR's, or documentation, if any
* [x] Resolved all conflicts, if any
* [x] Rebased your branch PR on top of the latest upstream `main` branch
* [x] Attached at least one of the following labels to the PR: `[chore, ci, docs, feat, fix, test]`
* [x] [Signed all commits using a GPG signature](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification#gpg-commit-signature-verification)

**To recreate commits with GPG signature** `git fetch upstream && git rebase --force --gpg-sign upstream/main`
_______________________________________________

Related to #1110 

## Description of the change:

Upgraded Topology View to Patternfly 5.

## Motivation for the change:

These changes are part of on-going efforts to upgrade to PF5.

## Others

Some PF5 dependencies must be bumped to latest (`v5.3.0`) that includes the fixes for the broken `TopologyControlBar` (previously showing tooltip inline).

image

### CODE REVIEW

**Comment by tthvo** [comment]
Label/Annotation filtering here is also affected by #1311. I can do a follow-up commit/PR after that issue is fixed :D

**Comment by andrewazores** [comment]
@tthvo hang on to your local copy of `pf5` if you can, or if you have a mirror of it on your fork. Not sure if I just screwed something up rebasing the upstream/pf5 onto upstream/main here.

**Comment by tthvo** [comment]
Sure, I just rebased yet. Still got the `pf5` copy here in locally :D

**Comment by andrewazores** [comment]
upstream/pf5 should be restored I think, please check it vs your local one. If it's correct I'll try the rebase onto main again.

**Comment by tthvo** [comment]
Confirmed it's correctly restored now. Both local and upstream pointing to correct commit :D

**Comment by andrewazores** [comment]
upstream/pf5 updated. I just merged main into it instead of trying to deal with the rebase.

**Comment by tthvo** [comment]
> upstream/pf5 updated. I just merged main into it instead of trying to deal with the rebase.

Nice! Thanksss! I rebased this PR now. Changes in #1312 is ported on this branch so filters should be working too :D

**Comment by andrewazores** [comment]
The decorator/badges for nodes don't seem to appear:

image

ie I think this node should have the green badge to indicate that it has running recordings?

**Comment by tthvo** [comment]
> ie I think this node should have the green badge to indicate that it has running recordings?

Phew, should be fixed now. It is a bit more setup to get tooltip working than previously in PF4.

File: src/app/Topology/Toolbar/DisplayOptions.tsx [hunk 1]
```diff
@@ -18,13 +18,16 @@ import { RootState, topologyDisplayOptionsSetIntent } from '@app/Shared/Redux/Re
 import {
   Checkbox,
```
Comment (andrewazores) [inline src/app/Topology/Toolbar/DisplayOptions.tsx:19]: Looks like some cleanup to do here

Comment (tthvo) [reply src/app/Topology/Toolbar/DisplayOptions.tsx]: Done^^

**Review by andrewazores — APPROVED** [review]

**Pull request merged**